addon/mixins/disposable.ts

```diff
@@ -1,5 +1,5 @@
 import Mixin from '@ember/object/mixin';
```

Comment:
Can you add a deprecation to this mixin (likely by adding an `init` and issuing the deprecation)?

Reply:
Yes. I was thinking we should deprecate them along with this too.

Reply:
Want to do that here, or in a separate PR?

Reply:
Done